packages/garbo/src/resources/piraterealm.ts

```diff
@@ -0,0 +1,174 @@
+import { Outfit, step } from "grimoire-kolmafia";
+import { $item, $location, $monster, $stat, get, have, uneffect } from "libram";
+import {
+  Effect,
+  myBuffedstat,
+  myEffects,
+  numericModifier,
+  runChoice,
+  toEffect,
+  visitUrl,
+} from "kolmafia";
+import { GarboTask } from "../tasks/engine";
+
+import { GarboStrategy, Macro } from "../combat";
+import { freeFightOutfit, meatTargetOutfit } from "../outfit";
+import { globalOptions } from "../config";
+
+const eyepatch = $item`PirateRealm eyepatch`;
+
+function keepStatsLow(): void {
+  const stats = [$stat`Muscle`, $stat`Moxie`, $stat`Mysticality`];
+  const effects: Effect[] = Object.keys(myEffects()).map((effectName) =>
+    toEffect(effectName),
+  );
+
+  stats.forEach((stat) => {
+    if (myBuffedstat(stat) > 100) {
+      // Get effect names from myEffects and convert them to Effect instances
+      effects.forEach((ef) => {
+        // Check if the effect modifier includes the stat and not "meat"
+        if (
+          numericModifier(ef, "muscle") &&
+          !(numericModifier(ef, "meat drop") > 0)
+        ) {
+          uneffect(ef); // Remove the effect
+        }
+      });
+    }
+  });
```

Comment:
use for loops, not forEach

Reply:
that way you can `break` the loop (or `return` out of the function) when you get below the stat threshold